src/Widgets/DisplayWidget.vala

```diff
@@ -16,6 +21,10 @@ public class SysMonitor.Widgets.DisplayWidget : Gtk.Grid {
         icon = new Gtk.Image.from_icon_name ("computer-symbolic", Gtk.IconSize.MENU);
         cpu_label = new Gtk.Label ("CPU");
         ram_label = new Gtk.Label ("MEM");
+        network_down_label = new Gtk.Label ("DOWN");
```

Comment:
these two labels are nice, but because the network info is changing very fast and the values have from 0 to 4 digits, it doesn't look nice and is kind of visually disturbing when all the indicators move when network info changes. Maybe you can find a way to set an optimal fixed width of the label so that the possible values could fit in there. This is also referring to the PopoverWidget.

Reply:
The size is now fixed, the indicator does not move anymore. It's a lot better.